tests-node/unit/broccoli/translation-reducer/find-missing-translations-test.js

```diff
@@ -22,17 +20,6 @@ describe('findMissingTranslations', function () {
       ['en', ['bar']],
     ];
 
-    expect(findMissingTranslations(key, localeKeys, isTrue)).to.deep.equal(['en']);
-  });
-
-  it('ignores missing translations if not required', function () {
```

Comment:
Rewrite test as part of a Linter unit/integration test now that this logic doesn't live within `findLocalesMissingTranslation`